filter/clausefilter.go

```diff
@@ -11,21 +11,10 @@ import (
 	"github.com/AdRoll/baker"
 )
 
-// ClauseFilterDesc describes the ClauseFilter filter
-var ClauseFilterDesc = baker.FilterDesc{
-	Name:   "ClauseFilter",
-	New:    NewClauseFilter,
-	Config: &ClauseFilterConfig{},
-	Help:   "This filter lets you set a boolean expression (in s-expression format) that will be matched against all records and dropped if they don't match the expression.\nCheck the filter file for documentation what the format looks like.\n",
-}
+const clauseFilterHelpMsg = `
+This filter lets you set a boolean expression (in s-expression format) that will be matched against all records and dropped if they don't match the expression.
```

Comment:
It's not just on this filter, I think we should do a pass on all docs but IMHO we could avoid starting the doc help string with _this filter_. Also here there's the use of passive. propostion:
```suggestion
Discard records which do not match a clause given as a boolean S-expression. Check the filter documentation for some examples.
```
Since the filter uses a mini lisp-like language to specify the clause, it seems important to ask the user to look at the documentation (or add a link to the doc page on pkg.go.dev -> https://pkg.go.dev/github.com/AdRoll/baker/filter#ClauseFilter)

Reply:
Ok for rephrasing. I'm not convinced about the "take a look at" because it's the same documentation the user is reading (either the go file or the web page, as they are the same). So the link to the web page will only show the same text but in the browser, if the user is reading the go file (and a link to itself in case of the web page) :thinking: